pluginManager/src/PluginManager.cpp

```diff
@@ -181,7 +181,7 @@ extern "C" __declspec(dllexport) void beNotified(SCNotification *notifyCode)
 #ifdef ALLOW_OVERRIDE_XML_URL
             ::MessageBox(nppData._nppHandle, _T("The Plugin Manager is running as a special build that allows override of the XML download URL.  This version should be used for testing purposes ONLY."), _T("Plugin Manager"), MB_ICONEXCLAMATION);
 #endif
-            ::CreateThread(NULL, 0, (LPTHREAD_START_ROUTINE)startupChecks, NULL, 0, 0);
+            (void)::CreateThread(NULL, 0, (LPTHREAD_START_ROUTINE)startupChecks, NULL, 0, 0);

```

Comment:
Just to avoid cppcheck warnings:

```
        <error id="ignoredReturnValue" severity="warning" msg="Return value of function CreateThread() is not used." verbose="Return value of function CreateThread() is not used.">
            <location file0="pluginManager/src/PluginManager.cpp" file="pluginManager\src\PluginManager.cpp" line="184"/>
        </error>
        <error id="leakReturnValNotUsed" severity="error" msg="Return value of allocation function &amp;#039;CreateThread&amp;#039; is not stored." verbose="Return value of allocation function &amp;#039;CreateThread&amp;#039; is not stored.">
            <location file0="pluginManager/src/PluginManager.cpp" file="pluginManager\src\PluginManager.cpp" line="184"/>
        </error>
```